src/topology/category/Profinite.lean

```diff
@@ -68,6 +68,33 @@ rfl
 
 @[simp] lemma coe_comp {X Y Z : Profinite} (f : X ⟶ Y) (g : Y ⟶ Z) : (f ≫ g : X → Z) = g ∘ f := rfl
 
+/-- Any morphism of profinite spaces is a closed map. -/
+lemma is_closed_map {X Y : Profinite} (f : X ⟶ Y) : is_closed_map f :=
+λ C hC, is_compact.is_closed $ is_compact.image (is_closed.compact hC) f.continuous
+
+/-- Any bijection is an isomorphism. -/
+lemma is_iso_of_bijective {X Y : Profinite} (f : X ⟶ Y) (bij : function.bijective f) : is_iso f :=
```

Comment:
```suggestion
lemma is_iso_of_bijective {X Y : Profinite} {f : X ⟶ Y} (bij : function.bijective f) : is_iso f :=
```

Reply:
I prefer to leave `f` explicit, since it's sometimes nice to tell lean what `f` is and add a proof of bijectivity on the fly.